src/v/v8_engine/environment.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Vectorized, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/vectorizedio/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
+
+#include <libplatform/libplatform.h>
+
+#include <v8.h>
+
+namespace v8_engine {
+
+// This class contains environment for v8.
+// Can be used only one time per application.
+class enviroment {
```

Comment:
Maybe this doesn't need to be a class? There can just be two functions, 1. init_v8_env(ptr) and 2. destroy_v8_env(ptr)

Reply:
In the next version I am going to add executor for v8_instance. And this class will be contain an executor.